examples/nas/enas/utils.py

```diff
@@ -0,0 +1,27 @@
+import torch
+
+
+def accuracy(output, target, topk=(1,)):
```

Comment:
seems there is another `accuracy` function in dart? similar?

Reply:
The return format is different. Accuracy is likely to be an infrastructure provided by NNI in future.

Reply:
OK